cql3/expr/to_restriction.cc

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "expression.hh"
+
+namespace cql3 {
+namespace expr {
+::shared_ptr<restrictions::restriction> to_restriction(const expression& e,
+                                                       data_dictionary::database db,
+                                                       schema_ptr schema,
+                                                       prepare_context& ctx) {
+    throw std::runtime_error(fmt::format("Unimplemented! {}:{}", __FILE__, __LINE__));
+}
```

Comment:
Please fold the implementing patch and this one into each other.

Reply:
done